doc/guide/introduction/tutorial.xml

```diff
@@ -315,4 +329,52 @@
           <video xml:id="gsv-2023-jan-video" youtube="y-33Ube-Rhk"/>
       </figure>
     </section>
+
+  <section xml:id="quickstart-example">
+    <title>For Advanced Users New to PreTeXt</title>
+
+    <p>
+      This section is intended for users who have read
+      <xref ref="start-here"/>
+      and are already experienced
+      with other markup languages and command-line tools. If you're
```

Comment:
Or maybe "other (markup) language and/or command-line tools"?  Obviously that is very nitpicky.